Source/WebCore/css/CSSPseudoSelectors.json

```diff
@@ -1,4 +1,29 @@
 {
+    "documentation": {
+        "description": "This file is used to generate code for pseudo-classes and pseudo-elements.",
+        "fields": {
+            "aliases": "Legacy name aliases for the pseudo. They will serialize to their modern counterpart.",
+            "argument": [
+                "Specifies if the pseudo takes an argument, and whether that argument is required or optional.",
+                "The 'optional' value is for pseudos like `:host` which support both `:host` and `:host(argument)`.",
+                "The 'required' value is for pseudos which don't support the argument-free syntax like: `::highlight(name)`.",
+                "Omit this field if the pseudo never takes an argument."
+            ],
+            "comment": "Add a description of how the pseudo is used. Especially useful for non-standard or internal ones.",
+            "condition": "Compile-time `#if ENABLE()` condition for the feature.",
+            "settings-flag": "Settings flag queried from CSSSelectorParserContext. Note that the flag needs to be manually added there.",
+            "status": "Specifies the standardization and shipping state of the pseudo.",
```

Comment:
Should this document the possible values?

Reply:
Right now the only value that's used is non-standard, it'll probably require a bit more thought with engineering/evangelism if we ever want to add more values. This maybe should be changed to a "non-standard": true boolean if we're not planning to extend the value space.